packages/tasks/package.json

```diff
@@ -7,33 +7,36 @@
 	"publishConfig": {
 		"access": "public"
 	},
-	"main": "./dist/index.cjs",
-	"module": "./dist/index.js",
-	"types": "./dist/index.d.ts",
+	"sources": "./src/index.ts",
+	"main": "./dist/commonjs/index.js",
+	"module": "./dist/esm/index.js",
+	"types": "./dist/commonjs/index.d.ts",
```

Comment:
iiuc from https://github.com/isaacs/tshy#old-style-exports-top-level-main-module-types, we should set `tshy.module` and `tshy.main` to `false` (I don't think we care about node < 12?)